samples/python/agents/semantickernel/agent.py

```diff
@@ -93,18 +93,53 @@ class SemanticKernelTravelAgent:
     SUPPORTED_CONTENT_TYPES = ['text', 'text/plain']
 
     def __init__(self):
-        api_key = os.getenv('OPENAI_API_KEY', None)
-        if not api_key:
-            raise ValueError('OPENAI_API_KEY environment variable not set.')
-
-        model_id = os.getenv('OPENAI_CHAT_MODEL_ID', 'gpt-4.1')
+        # Check if Azure OpenAI should be used
+        use_azure = os.getenv('ENABLE_AZURE_OPENAI', 'false').lower() == 'true'
+        
+        # Get API key based on selected service type
+        if use_azure:
```

Comment:
All of this `if use_azure` and explicitly grabbing env vars isn't needed. Let's let the service grab what it needs - if something isn't configured properly, it wil fail with a message showing what's missing.

Reply:
Resolved.